File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { QuotesOnDesignSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await QuotesOnDesignSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- QuotesOnDesign SDK exists test

local sdk = require("quotes-on-design_sdk")

describe("QuotesOnDesignSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
